web/test/data/private-project.json

```diff
@@ -1,6 +1,23 @@
 {
+  "uid": "test-private-project",
   "type": "private",
   "title": "Demo private project",
   "description": "Demo private project description added",
-  "url": "https://www.pipelinedashboard.io"
+  "_url": "https://www.pipelinedashboard.io",
+  "logoUrl": "",
+  "repositories": ["19deb070-1603-4fea-a774-6da2f93253ce", "97b2389a-4163-48b3-9fa8-ad8eb7937647"],
```

Comment:
These `uid`s will specific to an environment. I think we have a new PR to improve these, so probably ok for now

Reply:
okay